frontend/src/components/TaskMap.module.scss

```diff
@@ -26,7 +26,39 @@ $ZOOM: var(--zoom, 1);
   cursor: pointer;
 }
 
+.centerIcon {
+  right: 0;
+  cursor: pointer;
+
+  &.infoOpen {
+    transition: right 0.2s;
+    position: absolute;
+    right: 350px;
+  }
+}
+
+.restartIcon {
+  right: 0;
+  cursor: pointer;
+
+  &.infoOpen {
+    transition: right 0.2s;
+    position: absolute;
+    right: 325px;
+  }
+}
+
 .taskmapInfo {
   width: 27px;
   height: 27px;
+  cursor: pointer;
+}
+
+.infoDrawer :global .MuiDrawer-paper {
+  margin-top: 111px;
+  padding-bottom: 111px; // Margin top pushes the bottom of the drawer out of the picture without padding.
```

Comment:
Latest push only changed this:

```
margin-top: 115px -> 111px;
padding-bottom: 115px -> 111px;
```

(if you look closely, the info-drawer is a few pixels lower than the background):
![image](https://user-images.githubusercontent.com/80753884/173783063-553141ce-1001-4940-81bd-7b5bdbdf7656.png)